platforms/stm32_common/pcd.c

```diff
@@ -203,6 +203,7 @@ void HAL_PCD_DisconnectCallback(PCD_HandleTypeDef *p_pcd)
 		SELF.p_usbd->pClasses[i]->DeInit(SELF.p_usbd, SELF.p_usbd->dev_config);
 }
 
+#if defined(STM32L433xx)
```

Comment:
This could probably be more generic if there is a define for that, probably any chip from the L4 family will need this

Reply:
AFAIK the alterna I’ve is a plethora of stm32lxxxx defines which seems bloaty